python/ql/test/experimental/dataflow/coverage/classes.py

```diff
@@ -0,0 +1,1226 @@
+  with_itruediv /= with_itruediv
+
+# object.__ifloordiv__(self, other)
+class With_ifloordiv:
+
+  def __ifloordiv__(self, other):
+    OK()
+    return self
+
+def test_ifloordiv():
+  with_ifloordiv = With_ifloordiv()
+  with_ifloordiv //= with_ifloordiv
+
+# object.__imod__(self, other)
+class With_imod:
+
+  def __imod__(self, other):
+    OK()
+    return self
+
+def test_imod():
+  with_imod = With_imod()
+  with_imod %= with_imod
+
+# object.__ipow__(self, other[, modulo])
+class With_ipow:
+
+  def __ipow__(self, other):
+    OK()
+    return self
+
+def test_ipow():
+  with_ipow = With_ipow()
+  with_ipow **= with_ipow
+
+# object.__ilshift__(self, other)
+class With_ilshift:
+
+  def __ilshift__(self, other):
+    OK()
+    return self
+
+def test_ilshift():
+  with_ilshift = With_ilshift()
+  with_ilshift <<= with_ilshift
+
+# object.__irshift__(self, other)
+class With_irshift:
+
+  def __irshift__(self, other):
+    OK()
+    return self
+
+def test_irshift():
+  with_irshift = With_irshift()
+  with_irshift >>= with_irshift
+
+# object.__iand__(self, other)
+class With_iand:
+
+  def __iand__(self, other):
+    OK()
+    return self
+
+def test_iand():
+  with_iand = With_iand()
+  with_iand &= with_iand
+
+# object.__ixor__(self, other)
+class With_ixor:
+
+  def __ixor__(self, other):
+    OK()
+    return self
+
+def test_ixor():
+  with_ixor = With_ixor()
+  with_ixor ^= with_ixor
+
+# object.__ior__(self, other)
+class With_ior:
+
+  def __ior__(self, other):
+    OK()
+    return self
+
+def test_ior():
+  with_ior = With_ior()
+  with_ior |= with_ior
+
+# object.__neg__(self)
+class With_neg:
+
+  def __neg__(self):
+    OK()
+    return self
+
+def test_neg():
+  with_neg = With_neg()
+  -with_neg
+
+# object.__pos__(self)
+class With_pos:
+
+  def __pos__(self):
+    OK()
+    return self
+
+def test_pos():
+  with_pos = With_pos()
+  +with_pos
+
+# object.__abs__(self)
+class With_abs:
+
+  def __abs__(self):
+    OK()
+    return self
+
+def test_abs():
+  with_abs = With_abs()
+  abs(with_abs)
+
+# object.__invert__(self)
+class With_invert:
+
+  def __invert__(self):
+    OK()
+    return self
+
+def test_invert():
+  with_invert = With_invert()
+  ~with_invert
+
+# object.__complex__(self)
+class With_complex:
+
+  def __complex__(self):
+    OK()
+    return 0j
+
+def test_complex():
+  with_complex = With_complex()
+  complex(with_complex)
+
+# object.__int__(self)
+class With_int:
+
+  def __int__(self):
+    OK()
+    return 0
+
+def test_int():
+  with_int = With_int()
+  int(with_int)
+
+# object.__float__(self)
+class With_float:
+
+  def __float__(self):
+    OK()
+    return 0.0
+
+def test_float():
+  with_float = With_float()
+  float(with_float)
+
+# object.__index__(self)
+class With_index:
+
+  def __index__(self):
+    OK()
+    return 0
+
+def test_index():
+  import operator
+  with_index = With_index()
+  operator.index(with_index)
+
+# make With_index subscriptable to test slicing
```

Comment:
This looks like it might be a "to do" comment, or perhaps I'm misreading it?

Reply:
The comment is actually wrong, the test is simple to implement.